processing/src/main/java/org/apache/druid/segment/join/HashJoinSegmentStorageAdapter.java

```diff
@@ -69,8 +70,25 @@
       final List<JoinableClause> clauses,
       final JoinFilterPreAnalysis joinFilterPreAnalysis
   )
+  {
+    this(baseAdapter, null, clauses, joinFilterPreAnalysis);
+  }
+
+  /**
+   * @param baseAdapter           A StorageAdapter for the left-hand side base segment
+   * @param baseFilter            A filter for the left-hand side base segment
+   * @param clauses               The right-hand side clauses. The caller is responsible for ensuring that there are no
```

Comment:
No what?

Reply:
copy-paste error. will fix it in another patch.